recipes/onnxruntime/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "onnxruntime" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/microsoft/{{ name }}
+  git_rev: v{{ version }}
```

and here you should use ``sha256``
```suggestion
  sha256: 538c03b8226d7906bfa5a6538f95994460433b5b3b5eacdea2e1c79cada06756
```